src/System.Private.CoreLib/shared/System.Private.CoreLib.Shared.projitems

```diff
@@ -1101,16 +1136,23 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\Globalization\Normalization.Unix.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Globalization\TextInfo.Unix.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Guid.Unix.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\IO\DriveInfoInternal.Unix.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\IO\FileStream.OSX.cs" Condition="'$(TargetsOSX)' == 'true'" />
     <Compile Include="$(MSBuildThisFileDirectory)System\IO\FileStream.Linux.cs" Condition="'$(TargetsOSX)' != 'true'" />
     <Compile Include="$(MSBuildThisFileDirectory)System\IO\FileStream.Unix.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\IO\Path.Unix.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\IO\PathInternal.Unix.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\IO\PersistedFiles.Unix.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\IO\PersistedFiles.Names.Unix.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)System\PasteArguments.Windows.cs" />
```

Comment:
Is intentional to include `PasteArguments.Windows.cs` on Unix?

I have noticed this because VS complains about same file included multiple times in the project:

`Warning		The file 'C:\coreclr\src\System.Private.CoreLib\shared\System\PasteArguments.Windows.cs' could not be added to the project.  There is already a link to 'C:\coreclr\src\System.Private.CoreLib\shared\System\PasteArguments.Windows.cs'. A project cannot have more than one link to the same file.	System.Private.CoreLib`

Reply:
Nope, I'll fix it.